src/main/kotlin/calculator/Operators.kt

```diff
@@ -0,0 +1,16 @@
+package calculator
+
+object Operators {
+    private val operators: MutableMap<String, Operator> = HashMap()
+
+    init {
+        operators["+"] = Operator.PLUS
+        operators["-"] = Operator.MINUS
+        operators["*"] = Operator.MULTI
+        operators["/"] = Operator.DIVIDE
+    }
```

Comment:
```suggestion
    private val OPERATOR = mapOf(
        PLUS.name to PLUS,
        MINUS.name to MINUS,
        MULTI.name to MULTI,
        DIVIDE.name to DIVIDE
    )
```

`MutableMap`에다가 이미 있는 `Operator`를 넣는다면, `MutableMap`가 가변이기 때문에 나중에 자칫 잘못하면, 다른 값이 넣어질 가능성이 있습니다.

이런 경우라면, 차라리 `MapOf()`를 사용해서 초기에 다 넣고 불변을 지켜주는게 어떤가요?

`OPERATORS` 인 경우 상수로 적어 놓은 이유는